src/main/resources/assets/twilightforest/lang/zh_tw.lang

```diff
@@ -367,6 +434,8 @@ item.yetiPlate.tooltip=冰凍攻擊者
 item.yetiLegs.tooltip=冰凍攻擊者
 item.yetiBoots.tooltip=冰凍攻擊者
 
+item.arctic_armor.tooltip=像皮革裝備一樣可染色
+
 entity.twilightforest:wild_boar.name=野豬
```

Comment:
All of these `:` in the lang keys were changed to `.` long ago.
https://github.com/TeamTwilight/twilightforest/blob/1.12.x/src/main/resources/assets/twilightforest/lang/en_us.lang#L439-L498